code/modules/mob/mob_grab_specials.dm

```diff
@@ -43,19 +43,50 @@
 		if(!bad)
 			to_chat(user, SPAN_NOTICE("[H]'s skin is normal."))
 
+/obj/item/grab/proc/slow_bleeding(mob/living/carbon/human/H, mob/user, var/obj/item/organ/external/bodypart)
+
+	if(bodypart.is_stump() || !bodypart)
+		to_chat(user, SPAN_WARNING("They are missing that limb!"))
+		return
+	else
+		visible_message(SPAN_WARNING("[user] starts putting pressure on [H]'s wounds to stop the wounds on \his [bodypart.name] from bleeding!"))
+		if(!do_mob(user, H, 50))//5 seconds
+			to_chat(user, SPAN_NOTICE("You must stand still to stop the bleeding."))
+			return
+		else
+			visible_message(SPAN_NOTICE("[user] finishes putting pressure on [H]'s wounds."))
+			for(var/datum/wound/W in bodypart.wounds)
+				W.current_stage++
+				W.bleed_timer -= 5
+	//do not kill the grab
+
+
+/obj/item/grab/proc/force_vomit(mob/living/carbon/human/target, mob/attacker)
+	//no check for grab levels
+	visible_message(SPAN_WARNING("[attacker] places a finger in [target]'s throat, trying to induce vomiting."))//ewwies
+	attacker.next_move = world.time + 10 //1 second, also should prevent user from triggering this repeatedly
+	if(do_after(attacker, 10, progress=0) && target)
```

Comment:
a) this needs higher time because lolforcevomiting people
b) you can forcevomit people with covered mouth